### PR TITLE
feat: [FT-64] Use token set as source

### PR DESCRIPTION
**Pull request by LiamMartens**

### CODE REVIEW

File: src/plugin/node.ts [hunk 1]
```diff
@@ -114,7 +115,7 @@ export function selectNodes(ids: string[]) {
 
 export async function updateNodes(
   entries: readonly NodeManagerNode[],
-  tokens: Map<string, TokenArrayGroup[number]>,
+  tokens: Map<string, TokenArrayGrou[number]>,
```
Comment (six7) [inline src/plugin/node.ts:118]: this feels like a mistake?

File: src/utils/convertTokensObjectToResolved.ts [hunk 2]
```diff
@@ -18,7 +19,13 @@ export default function convertTokensObjectToResolved(
   // Parse tokens into array structure
   const parsed = parseTokenValues(tokens);
   // Merge to one giant array
-  const merged = mergeTokenGroups(parsed, usedSets);
+  const merged = mergeTokenGroups(
+    parsed,
+    // @README as I understand this function is only used in the utils/transform file
+    // which in turn is only used for a local script -- in which case for now we do not
```
Comment (six7) [inline src/utils/convertTokensObjectToResolved.ts:25]: correct, the token-transformer has a way to exclude token sets which is fine for now. users basically first define all inputs and later define which sets should be excluded from output

**Review by six7 — CHANGES_REQUESTED** [review]
Just this one mini comment that feels like a mistake, other than this good to merge!

**Comment by LiamMartens** [comment]
@six7 great! I fixed the typing so we can go ahead and merge this so we can continue with the themes :+1:

**Review by six7 — APPROVED** [review]

**Comment by six7** [comment]
Nice! 🎉

**Comment by mstllc** [comment]
@LiamMartens what does this `Treat as source` option actually do? 😄

**Comment by six7** [comment]
> @LiamMartens what does this `Treat as source` option actually do? 😄 

It doesnt include the tokens in the actual output, like if you wouldn't want your global tokens to create styles for you, as all you care about might be just the theme specific semantic tokens

**Pull request merged**